<?xml version="1.0"?>
<launch>
	<arg name="rgb_topic" default="/camera/rgb/" />
	<remap to = "$(arg rgb_topic)/camera_info"
		from = "rgb/camera_info"/>
	<remap to = "$(arg rgb_topic)/image_rect_color"
		from = "rgb/image_rect_color" />

	<remap to = "/camera/depth/camera_info"
		from = "depth/camera_info"/>
	<remap to = "/camera/depth/image_raw"
		from = "depth_registered/image_rect" />

	<remap to = "/vision/yolo2/detections"
		from="detection" />

	<node name = "reconstructor" output= "screen" pkg = "hands_3d" type = "reconstruct">
	</node> 
</launch>
